### PR TITLE
Add release notes for 5.19.3

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds release notes for 5.19.3

## Motivation and Context
Support 5.19.3 patch release

### CODE REVIEW

**Comment by hillaryfraley** [comment]
@nikkixdev  I have a few questions about the `Edition` in version info:
- Should users be able to see it if they run `sensuctl version`? It wasn't there in my testing so I thought I'd ask.
- Should it be in the response for a GET request to the versions API? Again it wasn't there in my testing.
- Will this be in the Tessen log that users can view? If so I should add it to https://docs.sensu.io/sensu-go/latest/reference/tessen/#tessen-metrics-log-examples

**Comment by nikkictl** [comment]
@hillaryfraley the changes for `Edition` have been retargeted against master so they will ship in 5.20.0 rather than 5.19.3! @amdprophet can answer that question for you when he's back, but we don't have to worry about it right now 😄

**Comment by hillaryfraley** [comment]
Thank you Nikki!

**Review by nikkictl — COMMENTED** [review]
I think we need to reference the final changelogs here in order to get all the release notes:
**sensu-go:** https://github.com/sensu/sensu-go/blob/release/5.19/CHANGELOG.md#5193---2020-04-30
**sensu-enterprise-go:** https://github.com/sensu/sensu-enterprise-go/blob/release/5.19/CHANGELOG.md#5193---2020-04-30

**Comment by hillaryfraley** [comment]
@nikkixdev got this updated with the final changelog content. I'm not sure how well I picked out the major changes/fixes for the summary.

**Review by nikkictl — APPROVED** [review]
LGTM! We might have another change or so land in the meantime since we are now targeting Monday for the release. I'll leave it up to you if you'd rather sit on this PR or open a new one if more changes come in.

File: content/sensu-go/5.19/release-notes.md [hunk 1]
```diff
@@ -58,6 +59,40 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+- Error logs now include underlying errors encountered when fetching an asset.
+- The log level is now WARNING when an asset is not installed because none of the filters match.
+- Fixes a bug in the [web UI][135] that could cause labels with links to result in a crash.
+- Fixes a bug in the [web UI][135] that could cause the web UI to crash when using an unregistered theme.
+- Fixes a bug that could cause the backend to crash.
+- Fixes a bug in multi-line metric extraction that appeared in Windows agents.
+- Fixes an authentication bug that restarted the sensu-backend when agents disconnected.
+- Fixes a bug that meant check `state` and `last_ok` were not computed until the second instance of the event.
+- Fixes a bug that caused messages like "unary invoker failed" to appear in the logs.
+- Fixes several goroutine leaks.
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md:85]:
```suggestion
- Fixes several goroutine leaks.
- Fixes a bug that caused the backend to crash when the etcd client received the error "etcdserver: too many requests."
```

**Pull request merged**